src/bootstrap/lib.rs

```diff
@@ -436,10 +436,9 @@ impl Build {
             output(Command::new(&build.initial_rustc).arg("--version").arg("--verbose"));
         let local_release = local_version_verbose
             .lines()
-            .filter(|x| x.starts_with("release:"))
+            .filter_map(|x| x.strip_prefix("release:"))
             .next()
             .unwrap()
-            .trim_start_matches("release:")
```

Comment:
Can you make note of this slight behavior change in the commit description? That'll help track it down if it causes any bugs (though I don't think it should).

Reply:
I added a rough explanation.